webapp/src/app/shared/widgets/vec2-widget/vec2-widget.component.html

```diff
@@ -6,14 +6,12 @@
 	       [step]="step"
 	       [min]="minSize.x"
 	       [disabled]="!enableX"
-	       [ngModel]="settings[key].x"
-	       (input)="setVal('x', toInt($event.target.value))">
+	       [(ngModel)]="settings[key].x">
```

Comment:
vec2 widget still needs the toInt function

Reply:
Will fix.